admin/server/device_code.go

```diff
@@ -0,0 +1,241 @@
+package server
```

Comment:
Move this file and the handlers to the `admin/server/auth` package, and register the endpoints in `auth.RegisterEndpoints`. Then we keep all auth isolated in that package.

Reply:
I tried doing that earlier as well but when I do it, its not able to resolve `"github.com/rilldata/rill/admin/server"`

Reply:
Discussing in Slack